raft/node.go

```diff
@@ -140,20 +140,10 @@ type Peer struct {
 	Context []byte
 }
 
-// StartNode returns a new Node given a unique raft id, a list of raft peers, and
-// the election and heartbeat timeouts in units of ticks.
+// StartNode returns a new Node given configuration and a list of raft peers.
+// It ignores the given peer list in the given Config.
```

Comment:
No, MultiNode does the same thing as Node: pass an empty peer array to newRaft and append ConfChange entries to the log. Only tests pass peers directly to newRaft.